continuedev/continue
# feat: allow JetBrains users to index docs

## Description

Allows JetBrains users to index docs.

## Checklist


## Screenshots

[ "For visual changes, include screenshots." ]

## Testing

Perform these actions in IntelliJ

- Verify that preindexed docs are not shown in the docs dropdown
- Add a new docs (through the UI or config.json, doesn't matter) and wait for it to index
- Change your embeddings provider to transformers.js
- Verify that you get an error
- Change back to a valid  provider
- Confirm that reindexing is kicked off (by waiting for toast notifications in the IDE) and that it completes
- Confirm that the reindexing was successful by using the docs context provider
